addons/account/models/account_journal.py

```diff
@@ -131,11 +131,7 @@ def _get_default_account_domain(self):
         readonly=False, store=True,
         help="Check this box if you don't want to share the same sequence for invoices and credit notes made from this journal",
     )
-    payment_sequence = fields.Boolean(
-        string='Dedicated Payment Sequence',
-        compute='_compute_payment_sequence', readonly=False, store=True, precompute=True,
-        help="Check this box if you don't want to share the same sequence on payments and bank transactions posted on this journal",
-    )
+    payment_sequence = fields.Boolean(string='Dedicated Payment Sequence', compute='_compute_payment_sequence')
```

Comment:
Why do we still need this field?

Reply:
Technically no, but is a nice and easy way to get whether or not at least one payment method has a dedicated payment sequence _aka._ ```any(journal.outbound_payment_method_line_ids.mapped('payment_sequence'))
                or any(journal.inbound_payment_method_line_ids.mapped('payment_sequence'))```

Is it preferable that I remove it completely and adapt the code accordingly ?